src/tutorial/runtime/introduction.rst

```diff
@@ -185,51 +178,42 @@ The Cylc GUI
    interface (the Cylc GUI) which can be used for monitoring and
    interaction.
 
-   The Cylc GUI looks quite like ``cylc graph`` but the tasks are colour-coded
-   to represent their state, as in the following diagram.
-
-.. digraph:: example
-   :align: center
-
-   Waiting [color="#88c6ff"]
-   Running [style="filled" color="#00c410"]
-   Succeeded [style="filled" color="#ada5a5"]
-
-.. minicylc::
-   :align: center
-
-    a => b => c
-    b => d => f
-    e => f
+   The Cylc UI has different views you can use to examine your workflows.
```

Comment:
If you're using the GUI in the tutorials we still need to introduce the main task state icons somehow.

Reply:
Done